partner_communication/models/communication_job.py

```diff
@@ -260,7 +256,12 @@ def write(self, vals):
         if vals.get('need_call'):
             vals['state'] = 'call'
 
-        return super(CommunicationJob, self).write(vals)
+        super(CommunicationJob, self).write(vals)
+
+        if vals.get('body_html') or vals.get('send_mode'):
```

Comment:
I would do ` or vals.get('send_mode') == 'physical'` to only count if we change to print the letter.

Reply:
That's true! Thanks